test/run-xspec-tests.cmd

```diff
@@ -39,12 +39,17 @@ for %%I in (*.xspec) do (
     rem
     rem Run
     rem
-    "%COMSPEC%" /c ..\bin\xspec.bat "%%~I" > "%RESULT_FILE%" 2>&1
+    call :is_schematron "%%~nI%"

```

Comment:
This doesn't have adverse affect right now, but the last `%` should be removed: `"%%~nI"`

Thanks for all the housekeeping you've taken!